### PR TITLE
turn on strict

### PR DESCRIPTION
**Pull request by baer**
Since any code using this is assumed to be running on node or common JS we can safely use module strict mode!

CC/ @ryan-roemer

### CODE REVIEW

File: rules/eslint/strict/walmart.js [hunk 1]
```diff
@@ -3,6 +3,6 @@
 module.exports = {
   rules: {
     // require that all functions are run in strict mode
-    'strict': [2, "never"]
+    'strict': [2, "always"]
```
Comment (ryan-roemer) [inline rules/eslint/strict/walmart.js]: Per our chat, let's choose a default for "all JS", which I think is still `never`.

**Comment by ryan-roemer** [comment]
@baer -- I think we're looking at tweaking global us strict back to `never`. And maybe switching `use strict` on in merged configurations. (Of course, might help for posterity's sake to re-title the PR since it's now just the airbnb es6 fixes).

Rest LGTM.

**Comment by baer** [comment]
Something is wrong with this PR on the Girhub side. Locally this is what I see. I am going to close and re-open this.

```
ebaer:~/dev/open-source/eslint-config-defaults ± (feature/turn-on-strict ✓)
git diff develop
diff --git a/rules/eslint/strict/walmart.js b/rules/eslint/strict/walmart.js
index 36b0724..3955cc5 100644
--- a/rules/eslint/strict/walmart.js
+++ b/rules/eslint/strict/walmart.js
@@ -3,6 +3,6 @@
 module.exports = {
   "rules": {
     // require that all functions are run in strict mode
-    "strict": [2, "never"]
+    "strict": [2, "global"]
   }
 };
```

**Comment by baer** [comment]
strict mode seems like one of the few things that you would really want to make sure is on for any project whether it is function mode (mixed strict and non-strict or code that is not using modules) or global (node/commonjs) since it changes JavaScript itself to be more helpful in the event of programming errors. Why would you want this off by default?

**Pull request merged**